datafusion/physical-expr-common/src/expressions/mod.rs

```diff
@@ -15,4 +15,11 @@
 // specific language governing permissions and limitations
 // under the License.
 
+//! Defines physical expressions that can evaluated at runtime during query execution
+
+#[macro_use]
+pub mod binary;
 pub mod column;
+pub mod datum;
```

Comment:
I agree it makes sense to have the datum / scalars / cast stuff in physical expr common